lib/mi_bridges/driver.rb

```diff
@@ -94,7 +94,9 @@ def setup
     end
 
     def teardown
-      MiBridges::Driver::BasePage.new(snap_application).close
+      if Rails.env != "test"
```

Comment:
Is this actually about the environment, or is it about whether we want to be debugging the application?

Reply:
in this case, it was so the test would run. I think the Debugger page stops things if we want to view the state of affairs otherwise? Or do you think this should depend on an env var rather than environment?

Reply:
Sorry I always forget to do the "review" mode. Yeah, I think making it rely on the same ENV variable as the other stuff that does the pre-deploy would make sense?

Reply:
>  I think making it rely on the same ENV variable as the other stuff that does the pre-deploy would make sense?

What do you mean by "other stuff" in that sentence? Is that a separate ENV var that releasing depends on? I looked around but didn't see anything.

Reply:
ah wait, now I get what you were saying! Yes, totally makes sense to use `ENV["PRE_DEPLOY_TEST"]` here, for consistency. done at 7469836